contrib/kind.sh

```diff
@@ -147,6 +149,125 @@ fi
 
 API_IPV6=""
 if [ "$KIND_IPV6_SUPPORT" == true ]; then
+  # Collect additional IPv6 data on test environment
+  # TEST-BEGIN
+  ERROR_FOUND=false
+  RESTART_DOCKER=false
+  TMPVAR=`sysctl net.ipv6.conf.all.forwarding | awk '{print $3}'`
+  echo "net.ipv6.conf.all.forwarding is equal to $TMPVAR"
+  if [ "$TMPVAR" != 1 ]; then
+    if [ "$KIND_UPDATE_SYSTEM" == true ]; then
+      sudo sysctl -w net.ipv6.conf.all.forwarding=1
+    else
+      echo "RUN: 'sudo sysctl -w net.ipv6.conf.all.forwarding=1' to use IPv6."
+      ERROR_FOUND=true
+    fi
+  fi
+  TMPVAR=`sysctl net.ipv6.conf.all.disable_ipv6 | awk '{print $3}'`
+  echo "net.ipv6.conf.all.disable_ipv6 is equal to $TMPVAR"
+  if [ "$TMPVAR" != 0 ]; then
+    if [ "$KIND_UPDATE_SYSTEM" == true ]; then
+      sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0
+    else
+      echo "RUN: 'sudo sysctl -w net.ipv6.conf.all.disable_ipv6=0' to use IPv6."
+      ERROR_FOUND=true
+    fi
+  fi
+
+  DOCKER_DAEMON_FILE=/etc/docker/daemon.json
```

Comment:
since kind 0.8.0 this is no longer needed, KIND creates its own bridge with ipv6 enabled

Reply:
I couldn't get it work without. But I'll try again. Maybe it was something else and this was just a red-herring.

Reply:
Just pushed a test to remove the logic and add a check of the KIND version:
$ kind version
kind v0.7.0 go1.13.6 linux/amd64

Reply:
Leaving in until we can upgrade to KIND v0.8.1. There was a lot of flakiness with KIND v0.8.1 and shared gateway mode had multiple consistent failures.